src/main/java/salesforce/classic/WorkTypeForm.java

```diff
@@ -0,0 +1,40 @@
+package salesforce.classic;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.BasePage;
+
+public class WorkTypeForm extends BasePage {
+    @FindBy(id = "Name")
+    protected WebElement nameTxtBox;
+
+    @FindBy(id = "EstimatedDuration")
+    protected WebElement estimatedDurationTxtBox;
+
+    @FindBy(name = "save")
+    protected WebElement saveBtn;
+
+    public WorkTypeForm(final WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    protected void waitForPageLoaded() {
+        wait.until(ExpectedConditions.visibilityOf(nameTxtBox));
+    }
+
+    public void setNameTxtBox(final String workTypeName) {
+        nameTxtBox.sendKeys(workTypeName);
+    }
+
+    public void setEstimatedDurationTxtBox(final String estimatedDuration) {
```

Comment:
does this method only work for Worktype? maybe could be used for other features and be defined in other class?